submission/programs/adadas.r

```diff
@@ -122,8 +101,65 @@ adas3 <- derive_vars_merged(
   )
 
 
-## baseline information BASE/CHG/PCHG
+## ANL01FL
 adas4 <- adas3 %>%
+  mutate(diff = AWTARGET - ADY) %>%
+  restrict_derivation(
+    derivation = derive_var_extreme_flag,
+    args = params(
+      by_vars = exprs(USUBJID, PARAMCD, AVISIT),
+      order = exprs(AWTDIFF, diff),
+      new_var = ANL01FL,
+      mode = "first"
+    ),
+    filter = !is.na(AVISIT)
+  )
+
+################# Feedback from FDA on need to match Pilot 1:
```

Comment:
For the comments regarding the FDA feedback and why we're making these changes, I wonder if this needs to be included here since we're already capturing this conversation in git comments.

Reply:
Referring to Lines 118 - 122.